planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -106,6 +111,12 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner(const rclcpp::NodeOptions & n
 
     // parameters for trajectory
     traj_param_ = TrajectoryParam(this);
+
+    // transform
+    rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+    map_odom_tf_buffer_ = std::make_shared<tf2_ros::Buffer>(clock);
+    map_odom_tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*map_odom_tf_buffer_);
+    odom_path_ptr_ = std::make_shared<Path>();
```

Comment:
I don't quite understand the purpose of this object.

Reply:
Do you mean `odom_path_ptr_`? It was first created to store the transformed path in odom frame but later on I find out it is not needed and forgot to delete.